templates/house_view/mine_page.html

```diff
@@ -3,6 +3,6 @@
 <body>
   <div class="house_table-container">{% include 'house_view/house_table.html' %}</div>
   <div class="sidebar-container">{% include 'house_view/sidebar.html' %}</div>
-
+  <div class="btn-container">{% include 'house_view/btns.html' %}</div>
```

Comment:
I would change the location of the Add Expense button

Reply:
What does that mean?
@adiabramovitch

Reply:
fixed